openapi/paths/journal-entries@{id}@records@{journalRecordId}.yaml

```
@@ -0,0 +1,85 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - $ref: ../components/parameters/journalRecordId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Journal
+  summary: Retrieve journal record
+  operationId: GetJournalRecord
+  x-sdk-operation-name: getRecord
+  description: Retrieves a journal record with a predefined identifier.
```

nitpick: based on our style
```suggestion
  description: Retrieves a journal record with a specified ID.
```